pkgs/applications/science/biology/deeptools/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, pytest
+, numpy
+, numpydoc
+, scipy
+, py2bit
+, pybigwig
+, pysam
+, matplotlib
+, plotly
+, deeptoolsintervals
+}:
+
+buildPythonPackage rec {
```

if you move this out of python-modules
```suggestion
buildPythonApplication rec {
```